bottleneck/csrc/move_median.h

```diff
@@ -4,6 +4,16 @@
 #include <math.h>
 #include <assert.h>
 
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+#define inline __inline
+static __inline float __NAN() {
+  float value;
+  memset(&value, 0xFF, sizeof(value));
+  return value;
+}
+#define NAN __NAN()
```

Comment:
@kwgoodman what if we changed line 14 to:

```
const double NAN = __NAN()
const double NAN64 = __NAN()
const float NAN32 = __NAN()
```

This will avoid calling memset() repeatedly, as there will be only one NAN value.  Also, later down the line, we can turn move_median into a template so that it is fast for float32 as well.